admin/orders.php

```diff
@@ -215,7 +215,7 @@
       }
 
       $email_include_message = (isset($_POST['notify_comments']) && $_POST['notify_comments'] == 'on');
-      $customer_notified = (int)(isset($_POST['notify'])) ? $_POST['notify'] : '0';
+      $customer_notified = (int)(isset($_POST['notify']) ? $_POST['notify'] : '0');
```

Comment:
Thanks for the fix.

Please adjust it to move the `(int)` next to `$_POST`, and change the `'0'` to `0`.

Reply:
it was a toss up...